### PR TITLE
add docs for arm build

### PR DESCRIPTION
**Pull request by yxlao**
This change is [Reviewable](https://reviewable.io/reviews/intel-isl/open3d/2445)

### CODE REVIEW

File: docs/arm.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    ./bin/tests --gtest_filter="-*Reduce*Sum*"
+
+    # Install Open3D python package (optional)
+    make install-pip-package -j$(nproc)
+    python -c "import open3d; print(open3d)"
+
+    # Run Open3D GUI (available on when -DBUILD_GUI=ON)
+    ./bin/Open3D/Open3D
+
+
```
Comment (ssheorey) [inline docs/arm.rst:102]: Is there an `install` step too?
```suggestion
    # Install Open3D binaries
    make install
```

File: docs/arm.rst [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+Nvidia Jetson
+-------------
+
+Nvidia Jetson computers with 64-bit processor and OS are supported. You can
+compile Open3D with ``-DBUILD_CUDA_MODULE=ON`` and ``-DBUILD_GUI=ON`` and
+the Open3D GUI app should be functional.
+
```
Comment (ssheorey) [inline docs/arm.rst:109]: Should we remind users here that we support CUDA 10.1 only?

**Review by ssheorey — APPROVED** [review]
Have we tried the build on actual hardware (Jetson, Rasperry Pi) or only on simulator?

File: docs/arm.rst (hunk 2 above)
Comment (yxlao) [reply docs/arm.rst:109]: In fact, if we don't consider the ML ops, the repo can work with a much wider range of CUDA versions. In my case, it is `10.2`.

File: docs/arm.rst (hunk 1 above)
Comment (yxlao) [reply docs/arm.rst:102]: yes, will add and mark as optional

**Comment by yxlao** [comment]
> Have we tried the build on actual hardware (Jetson, Rasperry Pi) or only on simulator?

Yes, tested on Jetson AGX (GUI=ON and CUDA=ON) and Rasperry Pi 4 (GUI=OFF due to OpenGL ES).

File: docs/arm.rst (hunk 2 above)
Comment (ssheorey) [reply docs/arm.rst:109]: In that case, let's add:
```suggestion
We support CUDA v10.1, but other versions should work as well.
```

Comment (yxlao) [reply docs/arm.rst:109]: done, added

**Pull request merged**